common/src/event.rs

```diff
@@ -162,6 +158,8 @@ pub enum MessageWokenSystemReason {
     TimeoutHasCome,
     /// Message can no longer pay rent for holding in storage (see #646).
     OutOfRent,
+    /// Waiting init list has been removed, so we just wake all the messages from it.
```

Comment:
According to comment above (on `enum MessageWaitedSystemReason`), please don't introduce any new dumb variants for temp purposes.

You have variant `ProgramGotInitialized`, that used previously. We can use it with note that it's incorrect naming blah blah. And this variant needs deprecation as well, isn't it?

Reply:
> dumb

Rude

> According to comment above (on enum MessageWaitedSystemReason), please don't introduce any new dumb variants for temp purposes.

I don't see any comments you are talking about

> You have variant ProgramGotInitialized, that used previously. We can use it with note that it's incorrect naming blah blah.

Variant is still used. Why don't introduce new one? No breaking changes because of re-fetched metadata

Reply:
We need formalize allowance of breaking changes for types which structure can be obtained via metadata. Looks like `Event` is sensitive

Reply:
> but this way isn't preferable

So should `Event` be treated as sensitive to changes or not?

Reply:
yep, always